_posts/2024-02-18-how-its-made.md

```diff
@@ -0,0 +1,178 @@
+
+Inevitably, a larger audience is also better in finding bugs, which is why the maintenance releases exists.
+Maintenance releases are the X.1, X.2 and so on versions.
+They do not include new features over the major X.0 release, but fix whatever issues are found.
+
+Thanks to a lot of behind the scenes magic done by some wizards on the CI workflows, the most time-consuming part of any release is actually creating and publishing all the public-facing text and information.
+Compiling the binary and uploading the result to the various distribution platforms requires surprisingly few mouse clicks to trigger.
+This wasn't always the case though.
+When OpenTTD was still young and sites like GitHub were just not a thing, the various binaries for e.g. Windows, Linux or macOS were hand-compiled by different people, manually collected and uploaded.
+Often enough that meant that it literally took several days until a release could be downloaded for all platforms.
+Compared to that, the current automatic CI workflows really do feel like awesome magic.
+
+
+## Getting personal
+
+If you've been diligently reading till now, you might have noticed that haven't used the word "developer" so far.
+Instead, I've mostly talked about a generic someone.
+So what's up with that?
+
+![Contributors to OpenTTD]({% link /static/img/post_2024-02-18-how-its-made/contributors.png %}){:width="300px" style="float:right; padding:10px"}
+
+While many people think that the individuals with merge permissions are "the" developers of OpenTTD, this isn't really true at all.
+GitHub provides various statistics for each project, and lists 169 contributors to OpenTTD at the time of writing this.
+And this number is still much too low, as it is for example missing many contributors from before we were on GitHub or our language translators.
+These hundreds of individuals are the real developers of OpenTTD and the people with merge permissions are more like housekeeping or maintenance.
+
+When you read something like "Why haven't the devs included X yet?", it is very rarely because someone with merge permissions said "nope", even if the questions is often phrased to imply this.
+It is not included yet because nobody volunteered their time for it.
+
+And that really is the gist of it.
+OpenTTD is not backed by a big company or some other institution, but purely by volunteer work.
+OpenTTD depends on donations.
+This does include monetary donations needed to pay for infrastructure and services, like our website and all the automated systems we've described here.
+But probably even more important, it also includes *time* donations.
+And time is what is missing most of time (ahem).
+So the best way to get X included is to donate some of your time.
+
+OpenTTD would not have made close to 20 years if it weren't for the hundreds of people that chose to donate some of their time to the project.
```

Comment:
This transition is a bit odd, and doesn't "flow". 

Maybe just a new chapters `## In closing` or something, might do the trick.